src/import/firefox.ts

```diff
@@ -180,7 +180,11 @@ export function importFromFirefox(firefoxProfile: FirefoxProfile): Profile {
 
         if (!match) return null
 
-        if (match[2].startsWith('resource:') || match[2] === 'self-hosted') {
+        if (
+          match[2].startsWith('resource:') ||
+          match[2] === 'self-hosted' ||
```

Comment:
Isn't this second check redundant? I think this can just be:

```
if (match[2].startsWith('resource:') || match[2].startsWith('self-hosted:'))
```

Right?

Reply:
I wasn't sure about why the `match[2] == self-hosted` check was added. I hypothesized that it worked in the past because older versions of Firefox or just Firefox in specific situations would generate locations with `self-hosted` and without a colon and other information following. I suppose `match[2].startsWith('self-hosted')` (no colon) would work too, but it feels a little weird.

Reply:
Oh you're totally right. This is fine, sorry I misread